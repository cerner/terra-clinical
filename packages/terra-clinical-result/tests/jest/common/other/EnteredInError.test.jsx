import React from 'react';
/* eslint-disable-next-line import/no-extraneous-dependencies */
import { shallowWithIntl } from 'terra-enzyme-intl';
import EnteredInError from '../../../../src/common/other/_EnteredInError';

describe('EnteredInError', () => {
  it('should render a default EnteredInError', () => {
    const result = shallowWithIntl(<EnteredInError />).dive();
    expect(result).toMatchSnapshot();
  });
});
