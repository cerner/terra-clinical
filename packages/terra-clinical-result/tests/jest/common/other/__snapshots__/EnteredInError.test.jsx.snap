// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnteredInError should render a default EnteredInError 1`] = `
<span
  className="entered-in-error"
>
  <span
    aria-label="Terra.clinicalResult.statusInErrorAria"
  >
    Terra.clinicalResult.statusInError
  </span>
</span>
`;
